### DISCUSSION TITLE
Great job with this project

### DISCUSSION THREAD

**Discussion by arvidjohansen** [agusmakmun/django-markdown-editor#198] [Q&A] [answered]
Hey man,

I'm using this markdown editor for a couple of projects that I'm working on now and I just gotta say that I love it!

Also I just browsed through this repo for the funz and the way you structured your code gave me a lot of learning experience.

Your code is so god damn clean and pythonic that I will definitely remember it for the future.

Good job

**Comment by agusmakmun** [comment, marked as answer]
Hello @arvidjohansen, I'm really happy to hear that. 🙌 

But, before that, here I'm not alone, I'll give special thanks to our contributors as well who contribute to the code, it's like: @KayleeTheMech, @Xyene, @quantum5, @olymk2, @tomaszn and many others that I can't say one by one. Also anyone who using this package without exception.

I didn't expect if this package raised more than 1k forks, because usually the package will have more numbers on stars rather than forks.

We're really open about suggestion, criticism, contribution or anything that can make this package more better. \
But, as you know, sometime I didn't have enough time to manage this package, because of busy on my job. \
With that said, I am really sorry if sometimes late response on replying the issues, updating the package, or so on.

That's why we really helpful if anyone contributes to the code.

Thank you so much @arvidjohansen 👍 👍 👍
